include/zephyr/net/ieee802154.h

```diff
@@ -42,6 +42,8 @@ extern "C" {
 
 /* See IEEE 802.15.4-2006, section 7.2.1.4 */
 #define IEEE802154_BROADCAST_ADDRESS 0xFFFF
+#define IEEE802154_NO_SHORT_ADDRESS_ASSIGNED 0xFFFE
```

Comment:
Split the commit. It's too many things in one, hard to follow.
- one about the short address it 802.15.4
- one for context
- one about the association/disassociation stuff

Reply:
Fixed.